libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -326,6 +338,19 @@ public override bool Equals(object obj)
 
         public override int GetHashCode() => (Id, Content).GetHashCode();
 
+        public void InjectToExprssionFunction()
```

Comment:
>InjectToExprssionFunction [](start = 20, length = 25)

typo in expression name (missing E)
I would call this RegisterExpressionFunction(), that matches other functions we have called Register

Reply:
fixed